.travis.yml

```diff
@@ -5,13 +5,17 @@ android:
     - platform-tools
     - tools
     - build-tools-28.0.3
-    - android-28
```

Comment:
Also, you have to specify the API level and the emulator level, so please do not delete `-android-<level>`. You need to specify for both of these separately, so you will have to add these lines
`-android-<your_API_level>`
`-android-<your_emulator_level>`
I hope this helps :)

Reply:
`-  android-28`
`-  android-27`
Okay will add back.